src/uct/ib/base/ib_md.c

```diff
@@ -365,14 +367,38 @@ static ucs_status_t uct_ib_md_post_umr(uct_ib_md_t *md, struct ibv_mr *mr,
     mrin.pd                       = md->pd;
 
 #ifdef HAVE_EXP_UMR_NEW_API
+#ifdef HAVE_EXP_UMR_KSM
```

Comment:
for cleanup, let's remove #ifdef HAVE_EXP_UMR_NEW_API, and always assume it's the new API (and update ib.m4 accordingly)